test/test_dwarf_expr.py

```diff
@@ -53,8 +53,11 @@ def test_basic_sequence(self):
         self.assertEqual(self.visitor.dump_expr([0x1d, 0x1e, 0x1d, 0x1e, 0x1d, 0x1e]),
             'DW_OP_mod; DW_OP_mul; DW_OP_mod; DW_OP_mul; DW_OP_mod; DW_OP_mul')
 
-        self.assertEqual(self.visitor.dump_expr([0x08, 0x0f, 0xe0]),
-            'DW_OP_const1u: 15; DW_OP_GNU_push_tls_address')
+        self.assertIn(self.visitor.dump_expr([0x08, 0x0f, 0xe0]),
+                      ('DW_OP_const1u: 15; DW_OP_GNU_push_tls_address',
+                       'DW_OP_const1u: 15; DW_OP_lo_user',
```

Comment:
The formatting is a bit weird here. Could you just tack the closing `))` to the end of the last line?